docs/production/running.md

```diff
@@ -0,0 +1,89 @@
+Local chunk storage is experimental and shouldn’t be used in production.
+
+Cortex requires a scalable storage back-end for production systems.
+It is recommended you use chunk storage with one of the following back-ends:
+
+* DynamoDB/S3 (see [Cortex on AWS](./storage-aws.md))
+* BigTable/GCS
+* Cassandra (see [Cortex on Cassandra](./storage-cassandra.md))
+
+Commercial cloud options are DynamoDB/S3 and Bigtable/GCS: the advantage is you don't have to know how to manage them, but the downside is they have specific costs.
+
+Alternatively you can choose Apache Cassandra, which you will have to install and manage.
+Cassandra support can also be used with commecial Cassandra-compatible services such as Azure Cosmos DB.
+
+Cortex has an alternative to chunk storage: [block storage](../operations/blocks-storage.md).  Block storage is not ready for production usage at this time.
+
+## 2. Deploy Query Frontend
+
+The **Query Frontend** is the Cortex component which parallelizes the execution of and caches the results of queries.
+The **Query Frontend** is also responsible for retries and multi-tenant QoS.
+
+For the multi-tenant QoS algorithms to work, you should not run more than two **Query Frontends**.
+The **Query Frontend** should be deployed behind a load balancer, and should only be sent queries -- writes should go straight to the Distributor component, or to the single-process Cortex.
+
+The **Querier** component (or single-process Cortex) “pulls” queries from the queues in the **Query Frontend**.
+**Queriers** discover the **Query Frontend** via DNS.
+The **Queriers** should not use the load balancer to access the **Query Frontend**.
+In Kubernetes, you should use a separate headless service.
+
+To configure the **Queries** to use the **Query Frontend**, set the following flag:
+
+```sh
+  -querier.frontend-address string
+    Address of query frontend service.
+```
+
+There are other flag you can use to control the behaviour of the frontend - concurrency, retries, etc.
+See [Query Frontend configuration](../configuration/arguments.md#Query_Frontend) for more information.
+
+The **Query Frontend** can run using an in-process cache, but should be configured with an external Memcached for production workloads.
+The next section has more details.
+
+## 3. Setup Caching
+
+Correctly configured caching is important for a production-ready Cortex cluster.
+Cortex has many opportunities for using caching to accelerate queries and reduce cost.
+
+For more information, see the [Caching in Cortex documentation.](./caching.md)
+
+## 4. Monitoring and Alerting
+
+Cortex exports metrics in the Prometheus format.
+We recommend you install and configure Prometheus server to monitor your Cortex cluster.
+
+We publish a set of Prometheus alerts and Grafana dashboards as the [cortex-mixin](https://github.com/grafana/cortex-jsonnet).
+We recommend you use these for any production Cortex cluster.
+
+## 5. Write Ahead Log
```

Comment:
In other parts of the doc, we mention it as experimental, but here we suggest to enable the WAL for a production-grade setup. Aren't the two things contradictory?